service/index.js

```diff
@@ -40,12 +40,12 @@ const putEditProfile = (userId, first_name, last_name, age, snack, animal_type)
     });
 };
 
-const getFriendsList = (userId) => {
+const getFriendsList = (email) => {
   return pool
     .connect()
     .then((client) => {
-      const query = 'SELECT first_name, last_name, thumbnail FROM users WHERE first_name = $1'; // to update params
-      const values = [userId];
+      const query = 'SELECT first_name, last_name, thumbnail FROM users AS u INNER JOIN friendship AS fs ON u.email = fs.friendEmail WHERE fs.userEmail = $1';
```

Comment:
ahhh beginning to enter the land of scary query's, I'll trust your inner joins

Reply:
We can test this when we connect the front end. It's working so far lol